doc/releases/release-notes-4.1.rst

```diff
@@ -111,6 +111,8 @@ Boards & SoC Support
 * Added support for these boards:
 
    * :zephyr:board:`Raspberry Pi Pico 2 <rpi_pico2>`: ``rpi_pico2``
+   * :zephyr:board:`Adafruit QT Py ESP32-S3 <adafruit_qt_py_esp32s3>`: ``adafruit_qt_py_esp32s3``
+   * :zephyr:board:`Adafruit QT Py ESP32-S3 with PSRAM <adafruit_qt_py_esp32s3@psram>`: ``adafruit_qt_py_esp32s3@psram``
```

Comment:
@fabiobaltieri @kartben So looks `adafruit_qt_py_esp32s3@psram` isn't a valid target and fails in CI. Should I just put `adafruit_qt_py_esp32s3` in the reference and only have the `@psram` part at the end instead?

Reply:
Ah yes I missed that in your diff. zephyr:board should point to a valid name which obviously the second line isn't. I'd just keep the first line as-is. I don't think documenting all the various combinations of variants/revisions necessarily makes sense as people mostly care about "oh, cool, that *board* was added".
Plus, that list of new boards will effectively be auto-generated / cleaned up, closer to 4.1 actual release

Reply:
ok, I'll remove the second line.